linera-base/build.rs

```diff
@@ -4,6 +4,7 @@
 fn main() {
     cfg_aliases::cfg_aliases! {
         web: { all(target_arch = "wasm32", feature = "web") },
+        chain: { all(target_arch = "wasm32", not(web)) },
```

Comment:
Or `onchain`?

Reply:
I considered it but it seemed to clash weirdly with `web` (`onweb`?).

I'd really like to create a [custom target](https://docs.rust-embedded.org/embedonomicon/custom-target.html) at some point, which I think obviates this.